protocol/x/clob/keeper/orders.go

```diff
@@ -454,6 +454,17 @@ func (k Keeper) PlaceStatefulOrdersFromLastBlock(
 
 		orderPlacement, exists := k.GetLongTermOrderPlacement(ctx, orderId)
 		if !exists {
+			// Error log if this is a conditional orders and it doesn't exist in state, since it
+			// can't have been canceled.
+			if orderId.IsConditionalOrder() {
```

Comment:
Conditional orders should always exist in state, so error logging here if they don't exist

Reply:
Also might be good to assert `IsConditionalOrderTriggered` is True. Since the conditional order has to be triggered at this point in time. It is possible to read out from untriggered state in`GetLongTermOrderPlacement`.

Reply:
^ should definitely assert that the conditional order is triggered